data/core/oos.md

```diff
@@ -19,7 +19,7 @@
 ## say greet outside the flows
 * greet
     - action_greet_user
-* greet OR enter_data{"name": "akela"}
```

Comment:
why did we remove that?

Reply:
it was conflicting with story 8:
```
## story number 8
* greet
    - action_greet_user
* enter_data
    - utter_not_sure   <!-- predicted: action_greet_user -->
```
which i've just realized is because I changed "name" to unfeaturized to use it in the form. I'll change back to separate slots for person_name(unfeaturized, for sales form) and name(text, for saying greet outside the flows)